dotCMS/src/test/java/com/dotmarketing/db/DataSourceStrategyProviderTest.java

```diff
@@ -0,0 +1,128 @@
+package com.dotmarketing.db;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import com.liferay.util.SystemEnvironmentProperties;
+import com.tngtech.java.junit.dataprovider.DataProvider;
+import com.tngtech.java.junit.dataprovider.DataProviderRunner;
+import com.tngtech.java.junit.dataprovider.UseDataProvider;
+import com.zaxxer.hikari.HikariDataSource;
+import javax.sql.DataSource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+
+/**
+ * Tests for {@link DataSourceStrategyProvider}
+ * @author nollymar
+ */
+@RunWith(DataProviderRunner.class)
+public class DataSourceStrategyProviderTest {
+
+    @DataProvider
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit 4 tests that execute tests should use the @Test annotation, JUnit 5 tests should use @Test, @RepeatedTest, @TestFactory, @TestTemplate or @ParameterizedTest](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5026767)